lib/chef/resource/macports_package.rb

```diff
@@ -16,10 +16,11 @@
 # limitations under the License.
 #
 
+require 'chef/resource/package'
+
 class Chef
   class Resource
     class MacportsPackage < Chef::Resource::Package
-      provides :package, os: "darwin"
```

Comment:
here you need to still provide package and the resolve the conflict later with the explicit priority setting.

facebook uses this, its an external API that both macports and homebrew declare 'provides' package on darwin.